.github/workflows/python_tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Python Test
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9, '3.10']
```

Comment:
Should '3.10' be in quotes or does it not matter?

Reply:
It does not **seem** to matter.  I copied that bit out of the comparable pyciemss and wondered the same thing.

Reply:
On second thought, i think its because 3.10 == 3.1 in numbers BUT we really mean a semantic version.